core/validator.go

```diff
@@ -31,6 +33,67 @@ func NewChunkValidator(enc Encoder, asgn AssignmentCoordinator, cst ChainState,
 	}
 }
 
+// preprocessBlob for each Quorum
```

Comment:
Can we expand this comment and make the function name clearer? It's not preprocessing so much as it is doing some header validation. Would `validateBlobMinusChunks` be appropriate?

Reply:
I can't tell what `validateBlobMinusChunks` means by looking at it, @jianoaix @ian-shim any other suggestions?

Reply:
`validateBlobQuorum`? It's primary validation of a quorum for a blob, secondary returning some needed info (better to comment this out)